.gitignore

```diff
@@ -10,3 +10,9 @@ node_modules
 /*/*/es
 /*/*/build
 /*/*/coverage
+addons/styled-components/.DS_Store
+addons/.DS_Store
+yarn.lock
+yarn-error.log
+yarn.lock
+yarn.lock
```

Comment:
Duplicate ignore entries.

.DS_Store should arguably be in your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).